src/app/screens/LNURLChannel/index.tsx

```diff
@@ -55,7 +63,10 @@ function LNURLChannel() {
         throw new Error(`Failed to call callback: ${callbackResponse.message}`);
       }
 
-      setSuccessMessage(`${t("success")}. ${details.k1} ${nodeId}`);
+      setSuccessMessage(
+        t("success", { name: origin ? origin.name : details.domain })
+      );
+      setResponse(`${details.k1} ${nodeId}`);
```

Comment:
Those are some internal information I don't thinks this is useful for the user.

Let's only show the success message and no response details.

(it's enough to say "Channel request sent successfully to {{name}}" )

Reply:
Makes sense, done!